protocol/messenger_settings_test.go

```diff
@@ -0,0 +1,109 @@
+package protocol
+
+import (
+	"context"
+	"testing"
+
+	"go.uber.org/zap"
+
+	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
+	"github.com/status-im/status-go/eth-node/crypto"
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/multiaccounts/common"
+	"github.com/status-im/status-go/protocol/encryption/multidevice"
+	"github.com/status-im/status-go/protocol/requests"
+	"github.com/status-im/status-go/protocol/tt"
+	"github.com/status-im/status-go/waku"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestMessengerSettings(t *testing.T) {
+	suite.Run(t, new(MessengerSettingsSuite))
+}
+
+type MessengerSettingsSuite struct {
```

Comment:
I think you can inherit from BasMessengerTestSuite or similar

Reply:
@qfrank sorry i missed that comment, will resolve it